src/controllers/brandsContraller.ts

```diff
@@ -0,0 +1,18 @@
+// brandsController.ts
+import { Request, Response } from 'express';
+import { BrandsModel } from '../models/brands';
+
+
+export const getALLBrands = async (req:Request , res:Response) =>{
+    try {
+       
+        const Brands = await BrandsModel.findAll();
+        return res.status(200).json({ data: { message:"success" , Brands} })
```

Comment:
no need for message: "success" in the response, it adds no value to the client.
response could be simplified as  `.json(Brands)`

Reply:
I will update it